modules/sdk-coin-atom/src/lib/iface.ts

```diff
@@ -29,7 +29,7 @@ export interface TxData extends AtomTransaction {
 export interface AtomTransaction {
   type: string;
   signerAddress: string;
-  explicitSignerData: SignerData;
+  sequence: number;
```

Comment:
Seems to be breaking change?, mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))

Reply:
Updated the commit message for the breaking change commit!